projects/plugins/jetpack/_inc/lib/class.color.php

```diff
@@ -36,6 +36,8 @@ class Jetpack_Color {
 		 *        One of hex (default), rgb, hsl, int.
 		 */
 		public function __construct( $color = null, $type = 'hex' ) {
+			_deprecated_function( 'Jetpack_Color::__construct', 'jetpack-$$next-version$$' );
```

Comment:
```suggestion
			_deprecated_function( __METHOD__, 'jetpack-$$next-version$$' );
```
This generates the same results (shows the function names), could be a more consistent way to add the deprecation notices. Was there a different reason why you used the strings directly though?